codespell_lib/data/dictionary.txt

```diff
@@ -22061,8 +22068,11 @@ reasonnable->reasonable
 reasonnably->reasonably
 reassocition->reassociation
 reasssign->reassign
+reasy->ready, easy,
```

Comment:
I guess you _could_ also suggest `greasy`, but I'm not sure if that's too much of a stretch?